src/libraries/Microsoft.Bcl.TimeProvider/src/System/Threading/Tasks/TimeProviderTaskExtensions.cs

```diff
@@ -218,5 +218,46 @@ public static async Task<TResult> WaitAsync<TResult>(this Task<TResult> task, Ti
             return task.Result;
         }
 #endif // NET8_0_OR_GREATER
+
+        /// <summary>Initializes a new instance of the <see cref="CancellationTokenSource"/> class that will be canceled after the specified <see cref="TimeSpan"/>. </summary>
+        /// <param name="timeProvider">The <see cref="TimeProvider"/> with which to interpret the <paramref name="delay"/>. </param>
+        /// <param name="delay">The time interval to wait before canceling this <see cref="CancellationTokenSource"/>. </param>
+        /// <exception cref="ArgumentOutOfRangeException"> The <paramref name="delay"/> is negative and not equal to <see cref="Timeout.InfiniteTimeSpan" /> or greater than maximum allowed timer duration.</exception>
+        /// <returns><see cref="CancellationTokenSource"/> that will be canceled after the specified <paramref name="delay"/>.</returns>
+        /// <remarks>
+        /// The countdown for the delay starts during the call to the constructor. When the delay expires,
+        /// the constructed <see cref="CancellationTokenSource"/> is canceled if it has
+        /// not been canceled already.
+        /// If running on framework version prior to .NET 8.0, there is a constraint when invoking <see cref="CancellationTokenSource.CancelAfter(TimeSpan)"/> on the resultant object.
```

Comment:
XML doc comments don't care about whitespace. You need to wrap paragraphs in `<para></para>` tags if you want the content to appear on a new line.

For example: https://github.com/dotnet/aspnetcore/blob/fca80883d95102eeb96e6fe9a68b170d54a4614c/src/Http/Http.Features/src/IFormFeature.cs#L16-L23

Reply:
We use the [porting tool](https://github.com/dotnet/api-docs-sync) to port documentation from source to the dotnet-api-docs repo. I believe remarks sections do have some degree of whitespace significance there so `<para />` tags aren't all that common in dotnet/runtime source code.

cc @carlossanlop